src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingService.cs

```diff
@@ -82,7 +48,7 @@ public ParameterCapturingService(IServiceProvider services)
                     IpcCommand.StopCapturingParameters,
                     OnStopMessage);
 
-                ILogger serviceLogger = services.GetService<ILogger<DotnetMonitor.ParameterCapture.Service>>()
+                _logger = services.GetService<ILogger<DotnetMonitor.ParameterCapture.Service>>()
```

Comment:
Save this for a separate PR... I don't see error handling for when we try to load into a application that doesn't have logging configured; I know this is very unlikely, but it looks like we might crash the startup of the application.

Reply:
Okay, I was missing the last catch and only saw the NotSupportException catch. Sorry!